bin/main.rb

```diff
@@ -0,0 +1,103 @@
+require_relative '../lib/player'
+
+require_relative '../lib/game'
+
+game = Game.new
+
+board = game.game_arr
+
+def welcome_screen(game, board)
```

Comment:
Why are you passing **game** or **board** in parameter when you declare them as global to that method (declaration at lines 5-7)?
It's Is not needed !!